src/main/java/com/adyen/service/resource/notification/DeleteNotificationConfiguration.java

```diff
@@ -0,0 +1,19 @@
+package com.adyen.service.resource.notification;
+
+import com.adyen.Service;
+import com.adyen.service.Resource;
+
+import java.util.Arrays;
+
+
+public class DeleteNotificationConfiguration extends Resource {
+
+    public DeleteNotificationConfiguration(Service service) {
+        super(service,
+              service.getClient().getConfig().getMarketPayEndpoint() + "/Notification/" + service.getClient().getMarketPayApiVersion() + "/deleteNotificationConfiguration",
```

Comment:
The endpoint for this call is /deleteNotificationConfigurations, please add the final 's'